CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Thank you for your interest in contributing to the 1Password load-secrets-action project 👋! Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Testing
```

Comment:
Since this action is a Node type action (i.e. executed using Node), we also have Typescript tests that can be run using `npm run test`.